pages/overview.vue

```diff
@@ -30,6 +30,7 @@ interface YAxisItem {
 const xAxisLabels: string[] = []
 const yAxisLabels: YAxisItem[] = []
 const pickedMoodsValues: number[] = []
+const pickedMoodsDescriptions: string[] = []
```

Comment:
Not the most elegant solution but in this instance the fastest and easiest. We probably should put moodValues and moodDescriptions in an array of objects.